Include/internal/pycore_atomic.h

```diff
@@ -46,6 +46,10 @@ typedef struct _Py_atomic_address {
     atomic_uintptr_t _value;
 } _Py_atomic_address;
 
+typedef struct _Py_atomic_ulong {
```

Comment:
Could you use either `_Py_atomic_load_uint32` or `_Py_atomic_load_uint64` instead of defining your own.
Since thread id is an `unsigned long`, you'll need to use a macro, but it seems better to use the pre-existing functions.
```C
#if SIZEOF_LONG == 8
#define _Py_atomic_load_ulong _Py_atomic_load_uint64
#elif SIZEOF_LONG == 4
#define _Py_atomic_load_ulong _Py_atomic_load_uint32
#else
#error "long must be 4 or 8 bytes in size"
#endif
```

Reply:
done